src/__itests__/App.browser.test.tsx

```diff
@@ -17,7 +17,7 @@ const getPath = (key: string) => {
 };
 
 const kialiPages = {
-  graph: `${getUrl()}/console/service-graph/all`,
+  graph: `${getUrl()}/console/graph/all`,
```

Comment:
Is 'Graph' really better terminology than ' Service Graph'?

Graph is general, Service-graph is specific. What happens when we add other graphs?